extensions/sql-migration/src/constants/strings.ts

```diff
@@ -180,32 +179,32 @@ export const SELECT_BLOB_CONTAINER = localize('sql.migration.blob.container.sele
 
 // integration runtime page
 export const IR_PAGE_TITLE = localize('sql.migration.ir.page.title', "Azure Database Migration Service");
-export const IR_PAGE_DESCRIPTION = localize('sql.migration.ir.page.description', "Azure Database Migration Service (DMS) orchestrates database migration activities and tracks their progress. You can select an existing DMS for Azure SQL target if you have created one previously or create a new one below.");
+export const IR_PAGE_DESCRIPTION = localize('sql.migration.ir.page.description', "Azure Database Migration Service (DMS) orchestrates database migration activities and tracks their progress. You can select an existing DMS as an Azure SQL target if you have created one previously, or create a new one below.");
 export const IR_PAGE_NOTE = localize('sql.migration.ir.page.note', "Note: DMS will run in your Azure subscription in the chosen resource group and does not incur any cost for running it.");
```

Comment:
I think DMS should not be abbreviated but spelled out as "Database Migration Service" There may be several instances of this to address.